lib/typescript/httpclient/model/Message.ts

```diff
@@ -35,7 +35,7 @@ export interface Message extends Content {
   metadata?: MessageMetadata;
 }
 
-export interface SuggestedReply {
+export interface SuggestedReply extends Content {
```

Comment:
would this type work with the Content type? 

`export interface Content {
  id: string;
  content: any;
}`